### ISSUE TITLE
Question about output

### ISSUE THREAD

**Issue by WangHeHehehehe** [xwjabc/hed#16]
Thank you for your code!
However,my output/epoch-#-train have 7 gray graphs,and the training process can be carried out without error reporting,looking forward to your reply!

**Comment by xwjabc** [comment]
Hmm, perhaps it is due to the environment settings. I wonder if you can provide a docker image that reproduces your issue and then I can try to find the reason behind it.

**Comment by WangHeHehehehe** [comment]
Thank you for your reply!
Using the current environment, I can complete the training and testing process of BSDS500, indicating that it should not be an environmental problem.
![图片](https://user-images.githubusercontent.com/58463086/71775847-72898f00-2fc2-11ea-8a76-a398af32952c.png)


Training set loading progress is 0%.

**Comment by WangHeHehehehe** [comment]
Thank you for your reply!
Training set loading progress is 0%, However, I can complete the training and testing process of bsds500 by using the current environment, indicating that it should not be an environmental problem.
![图片](https://user-images.githubusercontent.com/58463086/71775939-91d4ec00-2fc3-11ea-8673-0ef296fd566b.png)

**Comment by WangHeHehehehe** [comment]
![图片](https://user-images.githubusercontent.com/58463086/71777324-1c741600-2fd9-11ea-855b-8a0ebd4642d6.png)
This is the results of the epoch-#-train.

**Comment by xwjabc** [comment]
I see. I will try to run once today. If it can work at my side, I will provide my conda YAML file and you may create an environment based on it.

**Comment by xwjabc** [comment]
It is a bit strange that in your screenshot, your test set only has 5 images and your training set has 36 images. Ideally, there should be 200 images in test set and 28800 images in training set:
<img width="793" alt="Screen Shot 2020-01-05 at 2 21 37 PM" src="https://user-images.githubusercontent.com/16413829/71787025-1c6b1a80-2fc7-11ea-9b43-223db2b79d6d.png">

You may need to re-download the dataset and run the program.

**Comment by long123524** [comment]
My loss is very large, and 6 of the training results are all gray. Why is this?
![image](https://user-images.githubusercontent.com/73996375/124350006-9c0d9000-dc24-11eb-9778-037dfb8cc495.png)
![image](https://user-images.githubusercontent.com/73996375/124350009-a3cd3480-dc24-11eb-8071-bcb1bc2b890a.png)

**Comment by xwjabc** [comment]
@long123524 You may refer to my reply in https://github.com/xwjabc/hed/issues/36.
